script.js

```diff
@@ -1,4 +1,77 @@
+var SCISSORS = "scissors";
+var PAPER = "paper";
+var STONE = "stone";
+var REVERSE_SCISSORS = "reversed scissors";
+var REVERSE_PAPER = "reversed paper";
+var REVERSE_STONE = "reversed stone";
+var replayInstruction = "Type scissor, paper, or stone to play another round!";
+
+var computerRandom = function () {
+  var randomNum = Math.floor(Math.random() * 3);
+  if (randomNum == 0) {
+    return SCISSORS;
+  }
+
+  if (randomNum == 1) {
+    return PAPER;
+  }
+
+  return STONE;
+};
```

Comment:
Very very nice re-using of the strings stored in globals variables created from lines 1 to 3.